bin/nbp/nbplkup.c

```diff
@@ -64,7 +64,8 @@ int main(int ac, char **av)
     int i, c, nresp = 1000;
     struct at_addr addr;
     char *obj = NULL;
-    size_t obj_len;
+    char *type = NULL;
+    size_t obj_len, type_len;
```

Comment:
I have a mild preference for having a single declaration per line. Makes for a cleaner commit history, while arguably making the code more readable and less mistake prone.